clickhouse_search/models.py

```diff
@@ -798,6 +796,7 @@ class Meta:
 
 class BaseProjectGtStats(models.ClickhouseModel):
     project_guid = models.StringField(low_cardinality=True)
+    affected = models.Enum8Field(choices=[(1, 'A'), (2, 'N'), (3, 'U')], default='U')
```

Comment:
Do we want a default for this? You can do a no-default non-nullable column in django where it sets a default just for the migration and then going forward it needs to be explicit, which I think would be better here